deployer/src/algolia_internal_api.py

```diff
@@ -15,9 +15,9 @@ def get_endpoint(endpoint, params=''):
 def get_headers():
     token = environ.get('INTERNAL_API_AUTH')
 
-    app_id = environ.get('APPLICATION_ID_PROD')
-    admin_api_key = environ.get('API_KEY_PROD')
-    auth_token = b64encode(app_id + ":" + admin_api_key).replace('=',
+    app_id = environ.get('APPLICATION_ID_PROD').encode()
```

Comment:
Hard coding string is not a good practice 🤔
A file with const variable would be nice

Reply:
Definitely agree. This value is defined by constant environment variable. This change will impact a bunch of other files. Will keep it in our backlog and redesign it if needed